addons/gamification/models/gamification_challenge.py

```diff
@@ -74,7 +74,7 @@ class Challenge(models.Model):
         string="Responsible", help="The user responsible for the challenge.",)
     # members
     user_ids = fields.Many2many('res.users', 'gamification_challenge_users_rel', string="Users", help="List of users participating to the challenge")
-    user_domain = fields.Char("User domain", help="Alternative to a list of users")
+    user_domain = fields.Char("User domain", default='["&", ("groups_id", "ilike", "Internal User"), ("active", "=", True)]')  # Alternative to a list of users
```

Comment:
Hello @flch-odoo 

Can't we instead override the default_get to let us use a proper reference to the internal users group?
The ilike is a bit meh.

Should also account for the fact that the group may not exist, just to be safe.
(And in that case set the domain as empty? To avoid matching all users in the world?).

Cheers!

Reply:
Hello @tde-banana-odoo , 
After discussing this further at the Farm, and changes were pushed (`default_get`+assuming the group always exists otherwise everywhere anyway, but could change name), could we please have your opinion/approval for this solution?